### PR TITLE
add json parser

### PR DESCRIPTION
**Pull request by wtmount**

### CODE REVIEW

File: src/main/scala/jsonparser/JsonParser.scala [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+object JsonParser extends App {
+  val DbConfFilename = "db.properties"
+  val JsonFilename = "programming_languages.json"
+  implicit val formats = DefaultFormats
+
+  val db = new Properties()
+  db.load(Source.fromResource(DbConfFilename).bufferedReader())
+  var conn: Connection = _
+  try {
```
Comment (OstapKL) [inline src/main/scala/jsonparser/JsonParser.scala:20]:
```suggestion
  try {
    Class.forName("com.mysql.cj.jdbc.Driver")
```
Doesn't work for me without this line

**Review by OstapKL — COMMENTED** [review]
Good job! I like it. Thanks!

File: src/main/scala/jsonparser/JsonParser.scala (hunk 1 above)
Comment (wtmount) [reply src/main/scala/jsonparser/JsonParser.scala:20]: it's weird as Class.forName is not required since JDBC 4.0

**Comment by wtmount** [comment]
> Good job! I like it. Thanks!

Thank you)

File: src/main/scala/jsonparser/JsonParser.scala (hunk 1 above)
Comment (wtmount) [reply src/main/scala/jsonparser/JsonParser.scala:20]: anyway, added Class.forName

**Pull request merged**